static/app/components/onboardingWizard/progressHeader.tsx

```diff
@@ -14,6 +14,16 @@ type Props = {
 function ProgressHeader({allTasks, completedTasks}: Props) {
   const theme = useTheme();
 
+  let title: string, description: string;
+  const walkthrough = localStorage.getItem('new-walkthrough');
```

Comment:
could we abstract this into a utility function like `isDemoWalkthrough()` or something along those lines? I know we use this in a couple other places so it might be a tedious few changes, but to me I'm not quite sure what `new-walkthrough` is or the significance of the value since it is set from the `demo` repo.

Reply:
Additionally in that util function you could add a comment explaining that it comes from the sandbox

Reply:
oh that's a great idea, thanks David!!!